code/game/objects/items/devices/dummy_tablet.dm

```diff
@@ -12,6 +12,13 @@
 	linked_dummy = null
 	. = ..()
 
+/obj/item/device/professor_dummy_tablet/proc/is_adjacent_to_dummy(mob/user as mob)
```

Comment:
```suggestion
/// Checks if the provided mob is next to the dummy, returns TRUE if they are, and returns FALSE if they are not. Sends a message to the user if they are too far.
/obj/item/device/professor_dummy_tablet/proc/is_adjacent_to_dummy(mob/user)
```
`as user` is verb specific syntax, and we DMdoc new procs per https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md